### ISSUE TITLE
Any chance for 1.15.2?

### ISSUE THREAD

**Issue by DasFantatier** [Nuchaz/BiblioCraft#707]
Dear Mod-maker!

I don´t even know if this mod gets supportet any more, but is there a possibility that it will be compatible for 1.15.2?
I haven´t found any up to date information, so i hope it´s ok to ask here.

I really like this mod and we played it in 1.12 already.
Especially the wonderfull signs.

Greets, Das Fantatier.

**Comment by six6out** [comment]
True. Adding this to a server/client (no offense) is a pain in the ass.

--
JavaSight
Formerly lunatic-coding

**Comment by Peekofwar** [comment]
Probably not, due to the fact that it hasn't been updated to 1.13 or any version newer than said version, as 1.13 was a major overhaul. Someone somewhere even called 1.13 "the flattening."

My guess is that many parts of the mod would have to be modified or rewritten to be compatible with 1.13+.

**Comment by lehjr** [comment]
looks like the update is being worked on, just slow going: https://twitter.com/Mozzerello/status/1153040768602431489
